modules/library/main/src/main/java/org/geotools/styling/RasterSymbolizerImpl.java

```diff
@@ -255,7 +247,12 @@ public Expression getOpacity() {
      */
     @Deprecated
     public Expression getOverlap() {
```

Comment:
Why is getOverlap() deprecated; what is the replacement ... checking history this method has been deprecated since creation.

Looking at GeoServer docs this is marked as not implemented - http://docs.geoserver.org/latest/en/user/styling/sld/reference/rastersymbolizer.html#overlapbehavior

The specification matches the javadoc...